samples/cards/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@samples/cards",
+  "version": "0.0.0",
+  "license": "MIT",
+  "private": true,
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "scripts": {
+    "clean": "npx rimraf ./dist",
+    "build": "npx tsc",
+    "start": "node -r dotenv/config .",
+    "dev": "npx nodemon -w \"./src/**\" -e ts --exec \"node -r ts-node/register -r dotenv/config ./src/index.ts\"",
+    "dev:teamsfx": "npx env-cmd --silent -f .env npm run dev",
+    "dev:teamsfx:testtool": "npx env-cmd --silent -f .env npm run dev",
+    "dev:teamsfx:launch-testtool": "npx env-cmd --silent -f env/.env.testtool teamsapptester start"
+  },
+  "dependencies": {
+    "@microsoft/teams.apps": "*",
+    "@microsoft/teams.ai": "*",
+    "@microsoft/teams.dev": "*",
+    "@microsoft/teams.cards": "*"
+  },
+  "devDependencies": {
+    "@types/node": "^22.5.4",
+    "dotenv": "^16.4.5",
+    "nodemon": "^3.1.4",
+    "rimraf": "^6.0.1",
```

Comment:
rimraf not necessary since the script is using `npx`

Reply:
idek what rimraf was

Reply:
haha, recursive directory deleter